xapian-letor/api/featurelist.cc

```diff
@@ -132,6 +132,8 @@ FeatureList::create_feature_vectors(const Xapian::MSet & mset,
 	double wt = i.get_weight();
 	// Weight is added as a feature by default.
 	fvals.push_back(wt);
+	// Add a bias term.
+	fvals.push_back(1.0);
```

Comment:
I'm not at all sure I understand why a constant element in the feature vector is going to help any ranker, but even if it helps _some_ for some reason (such as an initialiser) I don't see why it would apply to _all_ rankers. This probably needs a bit more documentation to at least explain what's going on, and why it's always appropriate to add this element in.

This could and probably should have been a separate commit, as well.